test/rpc.hs

```diff
@@ -69,7 +75,7 @@ tests = testGroup "rpc"
     ]
   , testGroup "execution with remote state"
     -- execute against remote state from a ds-test harness
-    [ test "dapp-test" $ do
+    [ ignoreTestWindows $ test "dapp-test" $ do
```

Comment:
why don't these work on windows?

Reply:
Some of the git invocations that hevm did were failing and it was not obvious to me why. There's a comment with the error in the commit that added this ignore.

Reply:
could we just drop a source level comment here to make the reasoning here a little more accesible?